designs/virtualization/4.5/Scheduling/Tolerations/tolerations.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# VM Tolerations
```

Comment:
I added the same thought on the firmware PR and likely will be similar for others I review in the batch that are open :) Just thinking it might be good to have one page of documentation for the whole Scheduling section...watch do you think?

Reply:
It's a good idea. I think I better implement it after these PRs get merged

Reply:
+1...definitely merge all of these first and then reorg!